.github/workflows/ci.yml

```diff
@@ -116,13 +116,64 @@ jobs:
         if: always()
         with:
           path: ~/AppData/Local/Mozilla/sccache/cache
-          key: ${{ runner.os }}-sccache-${{ github.ref }}-${{ github.sha }}
+          key: ${{ runner.os }}-2022-sccache-${{ github.ref }}-${{ github.sha }}
       - name: test
         run: cd ..\b && ctest --timeout 300 --parallel %NUMBER_OF_PROCESSORS% -C Release --output-on-failure
         shell: cmd
       - name: linter
         run: cd ..\b && cmake --build . --target linter --config Release -j 8
         shell: cmd
+  windows_VS2019:
```

Comment:
Should we run both VS2019 and VS2022 jobs for every commit? We could extract one of them to a separate workflow like the gcc13 compatiblity job. We could create a cronjob for that (and for the gcc13 and python compatibility jobs as well while we are at it) and run them once a week for example.

Reply:
My plan is to
1. merge this PR
2. change the various Windows CI jobs to use VS 2022
3. remove this `windows_VS2019` job.

So this job should only be there for a few weeks.